packages/grafana-ui/src/components/ThemeColors/ThemeColors.tsx

```diff
@@ -31,7 +31,7 @@ const DemoBox: FC<DemoBoxProps> = ({ bg, border, children }) => {
           padding-bottom: 16px;
         `}
       >
-        {name}
+        {window.name}
```

Comment:
This looks funky to me. I guess this should be a `name` property maybe, but I cannot see anything like that in the props interface.

Reply:
Yeah, it's just another instance of what I mentioned in my other comment. `{name}` was actually referencing the global `name` property (which is the same as `window.name`). I assume this wasn't the intended behavior, but I changed it to `window.name` to satisfy the type checker as well as at least not _changing_ the behavior, even if it wasn't actually meant. Maybe someone else can clarify what the original intention was. @torkelo ?

Reply:
You can remove this whole div, not used, just results in an empty div https://developers.grafana.com/ui/latest/index.html?path=/story/docs-overview-themecolors--theme-colors-demo

Reply:
Do we want to keep the `16px` padding?